### PR TITLE
fix: not found index.js

### LINKED ISSUE #6764: vmui: UI for vlogs tries to load `dashboards/index.js`
### Describe the bug

vmui for VictoriaLogs tries to load `vmui/dashboards/index.js:1` when opened. It throws the error: `Failed to load resource: the server responded with a status of 404 ()`

### To Reproduce

Open https://play-vmlogs.victoriametrics.com/ and check browser console.

### Version

https://docs.victoriametrics.com/victorialogs/changelog/#v0280

### Logs

_No response_

### Screenshots

_No response_

### Used command-line flags

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by yincongcyincong**
### Describe Your Changes
fix #6764
the index.js file is for [this feature](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app/vmui#predefined-dashboards), the feature is just for victoriametrics. so the index.js is deleted in victorialogs.
i just add an empty index.js to fix it.

### CODE REVIEW

**Comment by Loori-R** [comment]
This is not quite the right approach. The correct solution would be to remove the mention of this script from `index.html` for the vmui logs build. An optimal solution might be to use different `index.html` files for separate builds. 
[Dashboards](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app/vmui#predefined-dashboards) are intended for predefined panels that are not used in the vmui logs build.

**Comment by yincongcyincong** [comment]
> The correct solution would be to remove the mention of this script from `index.html` for the vmui logs build. An optimal solution might be to use different `index.html` files for separate builds. [Dashboards](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app/vmui#predefined-dashboards) are intended for predefined panels that are not used in the vmui logs build.

thanks,bro ,i'll try to use this method.

**Comment by yincongcyincong** [comment]
@Loori-R bro, vm and vl have their own index.html (vm_index.html and vl_index.html). could you please review this code?

**Comment by hagen1778** [comment]
@Loori-R would you mind checking again?

File: app/vmui/packages/vmui/config-overrides.js [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+const path = require('path');
+
+// This will replace the default check
+if (process.env.REACT_APP_TYPE === "logs") {
+  const fileContent = fs.readFileSync(path.resolve(__dirname, 'public/vl_index.html'), 'utf8');
+  fs.writeFileSync(path.resolve(__dirname, 'public/index.html'), fileContent);
+} else {
+  const fileContent = fs.readFileSync(path.resolve(__dirname, 'public/vm_index.html'), 'utf8');
+  fs.writeFileSync(path.resolve(__dirname, 'public/index.html'), fileContent);
+}
```
Comment (Loori-R) [inline app/vmui/packages/vmui/config-overrides.js:14]: These changes will help remove code duplication and improve readability.

```suggestion
const pathIndexHTML = (() => {
  switch (process.env.REACT_APP_TYPE) {
    case 'logs':
      return 'public/victorialogs.html';
    case 'anomaly':
      return 'public/vmanomaly.html';
    default:
      return 'public/victoriametrics.html';
  }
})();

const fileContent = fs.readFileSync(path.resolve(__dirname, pathIndexHTML), 'utf8');
fs.writeFileSync(path.resolve(__dirname, 'public/index.html'), fileContent);
```

**Review by Loori-R — COMMENTED** [review]
Thank you for the pull request! I have a few comments:

- I suggest giving the HTML files clearer names so it's immediately obvious which version they are intended for. For example, `victorialogs.html`, `victoriametrics.html` and `vmanomaly.html`.
- Please add an HTML file for the `vmanomaly` version.

File: app/vmui/packages/vmui/config-overrides.js (hunk 1 above)
Comment (yincongcyincong) [reply app/vmui/packages/vmui/config-overrides.js]: thanks, bro. i have modify these code and add vmanomaly.html, please check it again.

**Review by Loori-R — APPROVED** [review]
LGTM! Thanks!

**Comment by Loori-R** [comment]
@hagen1778, could you merge it, please?

**Review by hagen1778 — APPROVED** [review]
Thanks for contribution!

**Pull request merged**